server/gateway.go

```diff
@@ -3078,18 +3093,18 @@ func (s *Server) startGWReplyMapExpiration() {
 				now := time.Now().UnixNano()
 				mapEmpty := true
 				s.gwrm.m.Range(func(k, _ interface{}) bool {
-					c := k.(*client)
-					c.mu.Lock()
-					for k, grm := range c.gwrm {
+					acc := k.(*Account)
+					acc.mu.Lock()
+					for k, grm := range acc.gwrm {
 						if grm.exp <= now {
-							delete(c.gwrm, k)
-							if len(c.gwrm) == 0 {
-								atomic.StoreInt32(&c.cgwrt, 0)
-								s.gwrm.m.Delete(c)
+							delete(acc.gwrm, k)
+							if len(acc.gwrm) == 0 {
+								atomic.StoreInt32(&acc.cgwrt, 0)
+								s.gwrm.m.Delete(acc)
```

Comment:
nil out acc.gwrm too?

Reply:
Concerned that we may get into a situation where we insert/remove often an then have to recreate the map all the times.